l10n_it_fatturapa/models/company.py

```diff
@@ -35,6 +35,7 @@ class ResCompany(models.Model):
         [
             ("Foglio_di_stile_fatturaordinaria_v1.2.2.xsl", "Fattura Ordinaria"),
             ("FoglioStileAssoSoftware.xsl", "AssoSoftware"),
+            ("FoglioStileSouthTyrol-bilingue.xsl", "Provincia di Bolzano"),
```

Comment:
```suggestion
            ("FoglioStileSouthTyrol-bilingue.xsl", "South-Tyrol"),
```
Senza entrare nella geografia, lo suggerisco solo perché è il nome usato in https://github.com/provbz/fatturapa-xsl-southtyrol/blob/31b4bdf9c6d5bc8fa0d79ea3e5e95fb003d56af3/README.md?plain=1#L1.
In generale, una qualsiasi stringa in inglese sarebbe preferibile.

Reply:
> Senza entrare nella geografia, lo suggerisco solo perché è il nome usato in https://github.com/provbz/fatturapa-xsl-southtyrol/blob/31b4bdf9c6d5bc8fa0d79ea3e5e95fb003d56af3/README.md?plain=1#L1. In generale, una qualsiasi stringa in inglese sarebbe preferibile.

Ora che vedo in effetti anche il foglio di stile standard andrebbe in inglese dato che compare nel .pot.

https://github.com/OCA/l10n-italy/blob/6f8f988bc0599f9300d0d32525a8ca76fc16a0ba/l10n_it_fatturapa/i18n/l10n_it_fatturapa.pot#L529